core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+
+- name: Download and unpack Helm binary
+  unarchive: 
+    src: "{{ apache_epirepo_path }}/files/helm-{{ helm_version }}-linux-amd64.tar.gz"
+    dest: "/usr/local/bin"
+    extra_opts:
+      - --strip=1
+      - --wildcards
+      - '*/helm'
+    remote_src: yes
+
+- name: Create Helm binary file symlink (RHEL) 
+  file:
+    src: "/usr/local/bin/helm"
+    dest: /usr/local/sbin/helm
+    state: link
+
+- name: Get list of Helm repos 
+  become: yes
+  shell: 'helm repo list | grep {{ helm_chart_repo_name }} || :'
+  register: helm_repo
+
+- name: Check if files exist in {{ apache_epirepo_path }}/helm-charts
+  become: yes
```

Comment:
I'd suggest using `true/false` instead of `yes/no` here, as `yes/no` is specific to ansible and openssh, but `true/false` you can find everywhere. Also please consider setting `become: true` at higher level, then you could remove repeating lines.